lib/orderbook/OrderBook.ts

```diff
@@ -65,17 +71,17 @@ class OrderBook extends EventEmitter {
   }
 
   /**
-   * Returns lists of buy and sell orders of peers sorted by price.
+   * Returns lists of buy and sell orders of peers
    */
-  public getPeerOrders = (pairId: string, maxResults: number): { [type: string]: orders.StampedOrder[] } => {
-    return this.getOrders(maxResults, this.peerOrders[pairId]);
+  public getPeerOrders = (pairId: string, maxResults: number): { [type: string]: orders.StampedPeerOrder[] } => {
+    return this.getOrders(maxResults, this.peerOrders[pairId]) as { [type: string]: orders.StampedPeerOrder[] };
   }
 
   /*
-  * Returns lists of the node's own buy and sell orders sorted by price.
+  * Returns lists of the node's own buy and sell orders
   */
-  public getOwnOrders = (pairId: string, maxResults: number): { [type: string]: orders.StampedOrder[] } => {
-    return this.getOrders(maxResults, this.ownOrders[pairId]);
+  public getOwnOrders = (pairId: string, maxResults: number): { [type: string]: orders.StampedOwnOrder[] } => {
+    return this.getOrders(maxResults, this.ownOrders[pairId]) as { [type: string]: orders.StampedOwnOrder[] };
   }
 
   private getOrders = (maxResults: number, orders: Orders): { [ type: string ]: orders.StampedOrder[]} => {
```

Comment:
It might be best just to use the `Map<string, StampedOwnOrder[]>` type since we've started using `Map` in more places in this PR.

Reply:
What bothered me about it is that it's hard to quickly understand what `type` means, because we have a few splitting dimensions for orders. 
But it's fine to leave it as is, because we're going to re-arrange this area anyway for #215.